src/pages/Home.js

```diff
@@ -42,7 +42,7 @@ const Home = () => (
         I can help you build a product, feature or website
         Look through some of my work and experience!
         If you like what you see and have a project you need coded,
-        try not to hestiate to contact me.
+        try not to hesitate to contact me.
```

Comment:
Kindly replace this paragraph with the one below;

`
Welcome to my portfolio!

I am excited to offer my skills to help you create your dream product, feature, or website. Please take a moment to browse through my previous work and experience. If you find yourself impressed with what you see, don't hesitate to reach out to me with your project. I'm looking forward to working with you!
`